Change test-QueuePlugin.mjs to test-Plugin.mjs and Tests Fail

I changes `test-QueuePlugin.mjs` to `test-Plugin.mjs` as part of #1084 but it causes tests to fail. For now I've just kept the file name as is.

Log of failing tests:

```
16 passing (1s)
  11 failing

  1) [workbox-background-sync] Queue
       addRequest
         should serialize the request and store it in IndexedDB:
     NotFoundError: IDBTransaction.objectStore(): Object store 'requests' does not exist
      at IDBTransaction.objectStore (/home/matt/Projects/Code/workbox/node_modules/shelving-mock-indexeddb/lib/mock.js:295:33)
      at Promise (/home/matt/Projects/Code/workbox/packages/workbox-core/_private/DBWrapper.mjs:117:1047)
      at new Promise (<anonymous>)
      at DBWrapper.transaction (/home/matt/Projects/Code/workbox/packages/workbox-core/_private/DBWrapper.mjs:117:154)
      at <anonymous>

  2) [workbox-background-sync] Queue
       addRequest
         should register to receive sync events for a unique tag:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:145:63)
      at <anonymous>

  3) [workbox-background-sync] Queue
       addRequest
         should invoke the requestWillEnqueue callback:
     NotFoundError: IDBTransaction.objectStore(): Object store 'requests' does not exist
      at IDBTransaction.objectStore (/home/matt/Projects/Code/workbox/node_modules/shelving-mock-indexeddb/lib/mock.js:295:33)
      at Promise (/home/matt/Projects/Code/workbox/packages/workbox-core/_private/DBWrapper.mjs:117:1047)
      at new Promise (<anonymous>)
      at DBWrapper.transaction (/home/matt/Projects/Code/workbox/packages/workbox-core/_private/DBWrapper.mjs:117:154)
      at <anonymous>

  4) [workbox-background-sync] Queue
       addRequest
         should support modifying the stored request via requestWillEnqueue:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:177:50)
      at <anonymous>

  5) [workbox-background-sync] Queue
       replayRequests
         should try to re-fetch all requests in the queue:

      AssertionError: expected 0 to equal 3
      + expected - actual

      -0
      +3

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:203:39)
      at <anonymous>

  6) [workbox-background-sync] Queue
       replayRequests
         should remove requests after a successful retry:

      AssertionError: expected 0 to equal 3
      + expected - actual

      -0
      +3

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:244:39)
      at <anonymous>

  7) [workbox-background-sync] Queue
       replayRequests
         should ignore (and remove) requests if maxRetentionTime has passed:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:272:42)
      at <anonymous>

  8) [workbox-background-sync] Queue
       replayRequests
         should keep a request in the queue if re-fetching fails:

      AssertionError: expected 0 to equal 2
      + expected - actual

      -0
      +2

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:300:33)
      at <anonymous>

  9) [workbox-background-sync] Queue
       replayRequests
         should re-register for a sync event if re-fetching fails:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:324:63)
      at <anonymous>

  10) [workbox-background-sync] Queue
       replayRequests
         should invoke all replay callbacks:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:344:50)
      at <anonymous>

  11) [workbox-background-sync] Queue
       replayRequests
         should support modifying the request via the requestWillReplay:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true

      at Context.<anonymous> (test/workbox-background-sync/node/lib/test-Queue.mjs:414:43)
      at <anonymous>
```

I'm not sure what the issue is or why they would affect each other, but would be good to rename the file to match the source file.

--- a/test/workbox-background-sync/node/lib/test-QueueStore.mjs
+++ b/test/workbox-background-sync/node/lib/test-QueueStore.mjs
@@ -14,40 +14,33 @@
 */
 
 import {expect} from 'chai';
-import clearRequire from 'clear-require';
 import {reset as iDBReset} from 'shelving-mock-indexeddb';
 import {DB_NAME, OBJECT_STORE_NAME} from
     '../../../../packages/workbox-background-sync/utils/constants.mjs';
+import {Queue} from
+    '../../../../packages/workbox-background-sync/Queue.mjs';
 import {QueueStore} from
     '../../../../packages/workbox-background-sync/models/QueueStore.mjs';
 import {DBWrapper} from '../../../../packages/workbox-core/_private/DBWrapper.mjs';
 import StorableRequest from
     '../../../../packages/workbox-background-sync/models/StorableRequest.mjs';
 
-let Queue;
-
 const getObjectStoreEntries = async () => {
   return await new DBWrapper(DB_NAME, 1).getAll(OBJECT_STORE_NAME);
 };
 
-
 describe(`[workbox-background-sync] QueueStore`, function() {
-  beforeEach(async function() {
-    // Clear Queue so the name map gets reset on re-import.
-    clearRequire('../../../../packages/workbox-background-sync/Queue.mjs');
+  const reset = () => {
+    Queue._queueNames.clear();
     iDBReset();
+  };
 
-    // Re-import Queue each time so the name map gets reset.
-    const imprt = await import(
-        '../../../../packages/workbox-background-sync/Queue.mjs');
-
-    Queue = imprt.Queue;
+  beforeEach(function() {
+    reset();
   });
 
   after(async function() {
-    // Clear Queue so the name map gets reset on re-import.
-    clearRequire('../../../../packages/workbox-background-sync/Queue.mjs');
-    iDBReset();
+    reset();
   });
 
   describe(`constructor`, function() {

--- a/test/workbox-background-sync/node/test-Plugin.mjs
+++ b/test/workbox-background-sync/node/test-Plugin.mjs
@@ -15,10 +15,8 @@
 
 import {expect} from 'chai';
 import sinon from 'sinon';
-import {Queue} from
-    '../../../../packages/workbox-background-sync/Queue.mjs';
-import {Plugin} from
-    '../../../../packages/workbox-background-sync/Plugin.mjs';
+import {Queue} from '../../../packages/workbox-background-sync/Queue.mjs';
+import {Plugin} from '../../../packages/workbox-background-sync/Plugin.mjs';
 
 describe(`[workbox-background-sync] Plugin`, function() {
   const sandbox = sinon.sandbox.create();

